src/ui/flutter_app/lib/shared/custom_drawer/src/widget.dart

```diff
@@ -113,6 +113,7 @@ class _CustomDrawerState extends State<CustomDrawer>
                 color: AppTheme.drawerDividerColor,
               ),
               ListView.builder(
+                controller: ScrollController(),
```

Comment:
I can't test this one :(

I guess it is for #388 If it does fix the issue please add `fixes #388` in the commit message or actually close the issue so we don't forget.

This is the sort of clean implementation I was looking for. Thanks xD

Reply:
No, that is a complex problem for me. What I solved was this problem:

Open Preference or Rule Settings or Personalization on Windows and scroll the mouse's wheel, triggering an exception, making it impossible to turn the page.

I will modify the Git commit message.